### DISCUSSION TITLE
Not able to deploy my website on github and showing deployment error

### DISCUSSION THREAD

**Discussion by romilp619** [vercel/next.js#52637] [Help]
### Summary

 npx --no-install next build
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_PAGES: true
  
warn  - Invalid next.config.js options detected: 
  - The value at .experimental has an unexpected property, images, which is not in the list of allowed properties (adjustFontFallbacks, adjustFontFallbacksWithSizeAdjust, amp, cpus, craCompat, disableOptimizedLoading, disablePostcssPresetEnv, esmExternals, appDir, allowMiddlewareResponseBody, externalDir, fallbackNodePolyfills, forceSwcTransforms, fullySpecified, gzipSize, incrementalCacheHandlerPath, isrFlushToDisk, isrMemoryCacheSize, largePageDataBytes, legacyBrowsers, manualClientBasePath, modularizeImports, newNextLinkBehavior, nextScriptWorkers, optimizeCss, optimisticClientCache, outputFileTracingRoot, pageEnv, profiling, proxyTimeout, runtime, serverComponentsExternalPackages, transpilePackages, scrollRestoration, sharedPool, skipMiddlewareUrlNormalize, skipTrailingSlashRedirect, sri, swcFileReading, swcMinify, swcMinifyDebugOptions, swcPlugins, swcTraceProfiling, urlImports, enableUndici, workerThreads, fontLoaders, webVitalsAttribution, mdxRs, turbotrace).

See more info here: https://nextjs.org/docs/messages/invalid-next-config
warn  - You have enabled experimental feature (images) in next.config.js.
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

warn  - No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

info  - Linting and checking validity of types...
info  - Creating an optimized production build...
Failed to compile.

./NFTDetailPage/NFTDescription/NFTDescription.jsx
Module not found: Can't resolve '../../Context/NFTMarketplaceContext' in '/home/runner/work/NFT/NFT/NFTDetailPage/NFTDescription'

Import trace for requested module:
./NFTDetailPage/NFTDescription/NFTDescription.jsx
./NFTDetailPage/NFTDetailsIndex.js
./NFTDetailPage/NFTDetailsPage.jsx

./componenets/Error/Error.jsx
Module not found: Can't resolve '../../Context/NFTMarketplaceContext' in '/home/runner/work/NFT/NFT/componenets/Error'

Import trace for requested module:
./componenets/Error/Error.jsx
./componenets/componentIndex.js

./componenets/HeroSection/HeroSection.jsx
Module not found: Can't resolve '../../Context/NFTMarketplaceContext' in '/home/runner/work/NFT/NFT/componenets/HeroSection'

Import trace for requested module:
./componenets/HeroSection/HeroSection.jsx
./componenets/componentIndex.js

./componenets/Navbar/Navbar.jsx
Module not found: Can't resolve '../../Context/NFTMarketplaceContext' in '/home/runner/work/NFT/NFT/componenets/Navbar'

Import trace for requested module:
./componenets/Navbar/Navbar.jsx

./pages/NFT-details.js
Module not found: Can't resolve '../Context/NFTMarketplaceContext' in '/home/runner/work/NFT/NFT/pages'

> Build failed because of webpack errors
Error: Process completed with exit code 1.

**Comment by icyJoseph** [comment]
You have a `.gitignore` file with this entry:

```
/Context
```

Which means that: `import { NFTMarketplaceContext } from "../../Context/NFTMarketplaceContext";` this import fails, because the `Context/NFTMarketplaceContext` is not being included in the git history.

You could also try to use the GitHub UI to try and find `NFTMarketplaceContext`, it is not there because you are ignoring it in the `.gitignore` file.
